Packs/LogRhythmRest/Integrations/LogRhythmRestV2/LogRhythmRestV2.py

```diff
@@ -1207,17 +1207,29 @@ def case_file_evidence_add_request(self, case_id, entry_id):
             file_bytes = file.read()
 
         file_content = file_bytes.decode('iso-8859-1')
-        content_type = mimetypes.guess_type(file_path)[0]
-
-        data = '-----------------------------\n' \
-               f'Content-Disposition: form-data; name="file"; filename="{file_name}"\n' \
-               f'Content-Type: {content_type}\n\n' \
-               f'{file_content}\n' \
-               '-----------------------------\n' \
-               'Content-Disposition: form-data; name="note"\n\n' \
-               '-------------------------------'
-
-        response = self._http_request('POST', f'lr-case-api/cases/{case_id}/evidence/file', data=data)
+        content_type = mimetypes.guess_type(file_name)[0]
+        demisto.debug(f'content-type of {file_name} is {content_type=}')
+
+        try:
+            data = '-----------------------------\n' \
+                   f'Content-Disposition: form-data; name="file"; filename="{file_name}"\n' \
+                   f'Content-Type: {content_type}\n\n' \
```

Comment:
Try to add it to a function, in order to avoid duplicate code

Reply:
good idea, will do